submissions/liswanisoLubinda/css/html/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <!-- Page title and style references here -->
+        <meta charset="UTF-8">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="../css/style.css">
+        <title>FoodBag</title>
+    </head>
+
+    <body>
+        <!--The main nav bar at the top of the page and its elements-->
+    <header class="header">      
+                <p>Food<span>Bag</span></p>
+                <ul class="top_bar">
+                    <li><a href="#">Cart</a></li>
+                    <li><a href="#">Login</a></li>
+                    <li><a href='#' class="sign_up">Sign up</a></li>
+                </ul>
+    </header>
+
+    <main class="main_body">
+        <section class="body">
+            <div class="body_content">
+                <h2>FoodBag</h2>
+                <p>Anything food, we have you covered.</p>
+            </div>
+        </section>
+
+        <!--Section and div to create form for customer to select their addresss-->
+        <section class="address">
+            <div class="customer_address">
+                <h3>Where are you?</h3>
+                <form action="#" method="post" enctype="multipart/form-data">
+                    <select name="address">
```

Comment:
in place of the select tag, please use datalist with an input field and a placeholder.

Reply:
Noted.